js/e2e/src/launch.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * pint-launch
+ */
+import fs from "fs";
+import findUp from "find-up";
+import path from "path";
+import { fork, ChildProcess, StdioOptions, spawn } from "child_process";
+
+/**
+ * Launch PINT locally
+ *
+ * @param stdio {StdioOptions}
+ * @returns {Promise<ChildProcess>}
+ */
+export async function local(stdio?: StdioOptions): Promise<ChildProcess> {
+    return fork("js/polkadot-launch", ["config.json"], {
```

Comment:
can't we youse polkadot-lauch directly?

Reply:
* The npm package of `polkadot-launch` is outdated
* If we use git url as a dependency 
  * The git repo of `polkadot-launch` is really big ( over 200MB ), because it contains binaries in its git history

Reply:
that's unfortunate. But as submodule we'd have to check it out anyways, so using the repo directly in package.json makes more sense to me

Reply:
I tried using `polkadot-launch` as a dependency before, and npm hardly can fetch `polkadot-launch` on my machine, let me try it on CI

Reply:
https://app.circleci.com/pipelines/github/ChainSafe/PINT/291/workflows/9f4b1a78-4d4a-4e37-a22c-58f4d2dcc97c/jobs/1492

Hardly can fetch